---
layout: paper
title: Association between triglycerides, known risk SNVs and conserved rare variation in SLC25A40 in a multi-ancestry cohort
image: /images/papers/Rosenthal-TG-2021.png
authors: Elisabeth A Rosenthal, David R Crosslin, Adam S Gordon, David S Carrell, Ian B Stanaway, Eric B Larson, Jane Grafton, Wei-Qi Wei, Joshua C Denny, Qi-Ping Feng, Amy S Shah, Amy C Sturm, Marylyn D Ritchie, Jennifer A Pacheco, Hakon Hakonarson, Laura J Rasmussen-Torvik, John J Connolly, Xiao Fan, Maya Safarova, Iftikhar J Kullo, Gail P Jarvik
year: 2021
ref: Rosenthal et al. 2021. BMC medical Genom.
journal: "BMC Medical Genomics"
pdf: /pdfs/papers/Rosenthal-TG-2021.pdf
doi: 10.1186/s12920-020-00854-2
---

# Abstract

Background<br>
DNA sequencing is increasingly incorporated into the routine care of cancer patients, many of whom also carry inherited, moderate/high-penetrance variants associated with other diseases. Yet, the prevalence and consequence of such variants remain unclear.<br>
Methods<br>
We analyzed the germline genomes of 10,389 adult cancer cases in the TCGA cohort, identifying pathogenic/likely pathogenic variants in autosomal-dominant genes, autosomal-recessive genes, and 59 medically actionable genes curated by the American College of Molecular Genetics (i.e., the ACMG 59 genes). We also analyzed variant- and gene-level expression consequences in carriers.<br>
Results<br>
The affected genes exhibited varying pan-ancestry and population-specific patterns, and overall, the European population showed the highest frequency of pathogenic/likely pathogenic variants. We further identified genes showing expression consequence supporting variant functionality, including altered gene expression, allelic specific expression, and mis-splicing determined by a massively parallel splicing assay.<br>
Conclusions<br>
Our results demonstrate that expression-altering variants are found in a substantial fraction of cases and illustrate the yield of genomic risk assessments for a wide range of diseases across diverse populations.
