<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Predicting functional effect of missense variants using graph attention neural networks</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">

	<!-- RSS feed -->
	<link rel="alternate" type="application/rss+xml" title="Fan Lab Research" href="http://xf-omics.github.io/feed.xml">

  <!-- Customized Bootstrap + Font Awesome + Solarized -->
  <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/DNA.png"/>

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>

</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light bg-navbar">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block align-top ml-0 mr-1" src="/images/DNA.png">
					Fan Lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link">
						
						<a class="mx-1" href="/blog/">news</a>
						</li>
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/papers/">papers</a>
						</li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/projects/">projects</a></li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">team</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container mt-4">

	<div class="row">
	<div class="col-lg-12">
		
		<img class="pull-left pad-right media-object d-none d-sm-block" src="/images/papers/Zhang-gMVP-2022.png">
		<img width=85 class="float-left pad-right d-block d-sm-none" src="/images/papers/Zhang-gMVP-2022.png">
		
		<div class="titlebox">
			<div class="title">
				Predicting functional effect of missense variants using graph attention neural networks
			</div>
			<p>
			<div class="smallhead">
				Haicang Zhang, Michelle S Xu, Xiao Fan, Wendy K Chung, Yufeng Shen
				2022.
				Nature Machine Intelligence
			</div>
		</div>
	</div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-lg-3">
		<div class="bigspacer"></div>
		<div class="glyphbox note">
			
			<div class="smallhead">
				PDF
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-file-text-o fa-fw"></i>
				<a class="off" href="/pdfs/papers/Zhang-gMVP-2022.pdf">Zhang-gMVP-2022.pdf</a>
			</div>
			<div class="bigspacer"></div>
			
			
			
			<div class="smallhead">
				DOI
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-external-link fa-fw"></i>
				<a class="off" href="https://doi.org/10.1038/s42256-022-00561-w">10.1038/s42256-022-00561-w</a>
			</div>
			<div class="bigspacer"></div>
			
			
		</div>
	</div>
	<div class="col-lg-8">
		<div class="post">
			<h1 id="abstract">Abstract</h1>

<p>Accurate prediction of damaging missense variants is critically important for interpreting a genome sequence. Although many methods have been developed, their performance has been limited. Recent advances in machine learning and the availability of large-scale population genomic sequencing data provide new opportunities to considerably improve computational predictions. Here we describe the graphical missense variant pathogenicity predictor (gMVP), a new method based on graph attention neural networks. Its main component is a graph with nodes that capture predictive features of amino acids and edges weighted by co-evolution strength, enabling effective pooling of information from the local protein context and functionally correlated distal positions. Evaluation of deep mutational scan data shows that gMVP outperforms other published methods in identifying damaging variants in TP53, PTEN, BRCA1 and MSH2. Furthermore, it achieves the best separation of de novo missense variants in neurodevelopmental disorder cases from those in controls. Finally, the model supports transfer learning to optimize gain- and loss-of-function predictions in sodium and calcium channels. In summary, we demonstrate that gMVP can improve interpretation of missense variants in clinical testing and genetic studies.</p>

		</div>
	</div>
	<div class="col-lg-1"></div>
</div>


	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>
